### ISSUE TITLE
Add Feature

### ISSUE THREAD

**Issue by GoodNames18** [lay295/TwitchDownloader#1252]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues?q=is%3Aissue) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

**### Task Queue**
Reverse queue so we don't need to scroll down if we have a lot to download.
**### Confirmation of file already exist**
add on the setting toggle on off, so i every time open the app. I dont need to choice remember this choice for this season. if I overwrite or rename
![Screenshot 2024-11-13 074352](https://github.com/user-attachments/assets/e15dcb35-a25a-4ead-82a5-0197bc4e6b4e)
**### VOD Download**
add name of the streamer to get info instead of vod id or both and history name if you type lor and i will show the lord streamer name.
when inputting name and gets info the 3 or 5 previous streamer vod will show and choose what vod to download.
I hope you understand what im saying

**Comment by Mishasama** [comment]
> when inputting name and gets info the 3 or 5 previous streamer vod will show and choose what vod to download.

![image](https://github.com/user-attachments/assets/87322035-2f51-4583-bff6-9067f1a7213c)
Are you talking something like about this?

**Comment by GoodNames18** [comment]
Yes but no trim feature or I don't see it and its better on the main tab of VOD Download on "VOD link/id" or add Name on it. 

This is a great feature if added and auto check the trim button selection
![Screenshot 2024-11-14 144401](https://github.com/user-attachments/assets/4e93fe15-b61c-42bd-a31d-bff525142f30)

**Comment by Mishasama** [comment]
So, do you mean TwitchLink?

**Comment by ScrubN** [comment]
> Reverse queue so we don't need to scroll down if we have a lot to download.

I think someone else also requested being able to sort the task queue, but its not a bad idea

> add on the setting toggle on off, so i every time open the app. I dont need to choice remember this choice for this season. if I overwrite or rename

This is already implemented

> add name of the streamer to get info instead of vod id or both and history name if you type lor and i will show the lord streamer name.

Someone else requested history and I agree that its a great idea, I just need to figure it out.

> Yes but no trim feature or I don't see it and its better on the main tab of VOD Download on "VOD link/id" or add Name on it.

Yeah I'm also not a huge fan of how limiting the enqueue window is. For now, you can use right-click to copy IDs from the Mass Downloader windows, then paste those in the VOD Download tab.
